examples/player/simid_player.js

```diff
@@ -144,8 +158,12 @@ class SimidPlayer {
     // The target of the player to send messages to is the newly
     // created iframe.
     playerDiv.appendChild(simidIframe);
-    // Set up css to overlay the SIMID iframe over the video creative.
-    simidIframe.classList.add('simid_creative');
+    // Set up css to overlay the SIMID iframe over the entire video creative
```

Comment:
Treat these comments as explanations to developers that might be reading this, move it right into the if statement.

Does this method even work of using css?  When you specify specifying a specific size how does this select the correct offsets etc.